src/components/LocationRow.tsx

```diff
@@ -17,14 +17,32 @@ function LocationRow(props: LocationRowProps): JSX.Element {
 
   if (error) {
     <View style={styles.wrapper}>
-      <Text>Not available...</Text>
+      <View style={styles.glassWrapper}>
+        <View style={styles.opacity}>
+          <View style={styles.left}>
+            <View>
+              <Text style={styles.header}>There was an error getting the forecast.</Text>
+            </View>
+          </View>
+        </View>
+        <BlurView blurAmount={25} blurType='light' />
+      </View>
     </View>
   }
 
   if (!forecast) {
     return (
       <View style={styles.wrapper}>
-        <Text>Loading...</Text>
+        <View style={styles.glassWrapper}>
```

Comment:
If I give a wrong URL here, the spinner hangs forever on the no-location screen

Reply:
Noted. Made change. Please have a look.